R/htmlwidgets.R

```diff
@@ -269,17 +274,25 @@ shinyRenderWidget <- function(expr, outputFunction, env, quoted) {
         instance$elementId, "\"; Shiny doesn't use them"
       )
     }
-    x <- .subset2(instance, "x")
     deps <- .subset2(instance, "dependencies")
     deps <- lapply(
       htmltools::resolveDependencies(deps),
       shiny::createWebDependency
     )
-    evals = JSEvals(x)
-    list(x = x, evals = evals, deps = deps)
+    payload = modifyList(createPayload(instance), list(deps = deps))
```

Comment:
`c(createPayload(instance), list(deps = deps))` looks more natural to me here compared to `modifyList()`.